ray-operator/apis/ray/v1/raycluster_types.go

```diff
@@ -75,6 +75,15 @@ type HeadGroupSpec struct {
 	// EnableIngress indicates whether operator should create ingress object for head service or not.
 	// +optional
 	EnableIngress *bool `json:"enableIngress,omitempty"`
+	// Resources specifies the resource quantities for the head group.
+	// These values override the resources passed to `rayStartParams` for the group.
```

Comment:
Can you also add comments here about the relationship between `resources` and resoruces at the container level?